source/vk/CMakeLists.txt

```diff
@@ -90,6 +90,11 @@ target_include_directories(VK PUBLIC
   ${CMAKE_CURRENT_SOURCE_DIR}/include)
 target_include_directories(VK SYSTEM PUBLIC
   ${CMAKE_CURRENT_SOURCE_DIR}/external/Khronos/include)
+
+# TODO: has some llvm dependencies via spirv/module.h. This should be looked at
```

Comment:
Just to check  - do you mean `spirv-ll/module.h` here?

Reply:
updated comment